### PR TITLE
Membership issue template

### PR DESCRIPTION
**Pull request by CalvinAllen**
## Description Of Change

Per committee meeting on 2021-11-03, it was decided to use an issue template for easier application to be part of the committee.

## Purpose Of Change

For ease of requesting membership to the membership WG

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=35) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=35) before we can accept your contribution.<br/><br/>:x: CalvinAllen [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/wg-membership?pullRequest=35)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/wg-membership?pullRequest=35) it.</sub>

File: README.md [hunk 1]
```diff
@@ -24,11 +24,8 @@ If you have an idea on how the membership offering could be enhanced or any issu
 
 ### Joining And Attending Membership Meetings
 
-- Clone this repo.
-- Open a branch called _github handle_/request-membership
-- Add your name and link to your GitHub handle in the `Members` section below.
-- Raise a PR with this change.
-- On seeing this [Bill Wagner](https://github.com/BillWagner) will merge and send you the next meetings invite.
+- [Create an issue](https://github.com/dotnet-foundation/wg-membership/issues/new/choose) using the "Membership" issue template
+- On seeing this [Bill Wagner](https://github.com/BillWagner) will accept the issue and close it, then send you the next meetings invite.
```
Comment (BillWagner) [inline README.md:28]: Gee, thanks. 😄

**Review by BillWagner — APPROVED** [review]
THanks again @CalvinAllen 

I'm glad you're joining the committee.

I'll :shipit: now.

File: README.md (hunk 1 above)
Comment (CalvinAllen) [reply README.md:28]: Well, you were already listed there, so I didnt want to change it.  More than happy to take some of that on myself, too. :)

**Pull request merged**